pysteps/verification/salscores.py

```diff
@@ -302,10 +302,10 @@ def _sal_l2_param(prediction, observation, thr_factor, thr_quantile, tstorm_kwar
     )
     obs_r = (
```

Comment:
please simplify the syntax by removing the unnecessary brackets and keeping it on one line, ie
```
obs_r = _sal_weighted_distance(observation, thr_factor, thr_quantile, tstorm_kwargs)
```

Reply:
It is don!